index.html

```diff
@@ -84,6 +84,7 @@ <h2>Privacy Protections Tests</h2>
 	<li><a href='./privacy-protections/query-parameters/'>Query Parameters</a></li>
 	<li><a href='./privacy-protections/local-port-scan/'>Local port scanning</a></li>
 	<li><a href='./privacy-protections/bounce-tracking/'>Bounce tracking</a></li>
+        <li><a href='./privacy-protections/request-blocklist/'>Request Blocklist</a></li>
```

Comment:
nit: can you fix the indentation here? Also is there something we can do to better differentiate between this test page and the request blocking test page? Like make it clear that this one is for visual/manual checking or something?